xcp_d/utils/bids.py

```diff
@@ -287,6 +298,10 @@ def collect_data(
     if len(densities) > 1:
         queries["bold"]["density"] = densities[0]
 
+    if input_type == "hcp" or input_type == "dcan":
+        queries["template_to_t1w_xform"]["from"] = "MNI152NLin2009cAsym"
+        queries["t1w_to_template_xform"]["to"] = "MNI152NLin2009cAsym"
```

Comment:
If you want to hardcode the space in, then I would recommend only looping through spaces above if `input_type` isn't hcp or dcan.

Reply:
Sorry, why did you resolve this?

Reply:
I think it doesn't loop through for temp_queries anymore